### PR TITLE
Add Orbit keyboard

### PR DESCRIPTION
**Pull request by ai03-2725**
Adds support for the Orbit Keyboard.

## Description

Adds support for the Orbit split ergonomic keyboard.
Due to the board having indicator LEDs on both sides, many edits had to be implemented to split_common files to sync layer and indicator status data. For this reason, the implementation provides its own copy of modified split_common files rather than using the default split_common.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* Add Orbit keyboard

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/ai03/orbit/matrix.c [hunk 1]
```diff
@@ -0,0 +1,328 @@
...
+  debounce(raw_matrix, matrix + thisHand, ROWS_PER_HAND, changed);
+
+  return 1;
+}
+
+uint8_t matrix_scan(void) {
+  uint8_t ret = _matrix_scan();
+
+  if (is_keyboard_master()) {
+    static uint8_t error_count;
```
Comment (mechmerlin) [inline keyboards/ai03/orbit/matrix.c]: Perhaps I'm missing something here. You're declaring a static `error_count` in line 306 and only incrementing it in line 309. `ERROR_DISCONNECT_COUNT` is defined to 5. I'm not seeing how error_count could ever equal more than 1.

File: keyboards/ai03/orbit/config.h [hunk 2]
```diff
@@ -0,0 +1,254 @@
...
+ *
+ * The options below allow the magic key functionality to be changed. This is
+ * useful if your keyboard/keypad is missing keys and you want magic key support.
+ *
+ */
+
+/* key combination for magic key command */
+#define IS_COMMAND() ( \
+    keyboard_report->mods == (MOD_BIT(KC_LSHIFT) | MOD_BIT(KC_RSHIFT)) \
+)
```
Comment (vomindoraan) [inline keyboards/ai03/orbit/config.h]: This is no longer required to be defined if it's using the default value (LShift+RShift). Please remove this `#define`, along with the comment and blank line above it (lines 141–145 as it currently stands).

File: keyboards/ai03/orbit/config.h [hunk 3]
```diff
@@ -0,0 +1,254 @@
...
+ * the keyboard. They are best used in combination with the HID Listen program,
+ * found here: https://www.pjrc.com/teensy/hid_listen.html
+ *
+ * The options below allow the magic key functionality to be changed. This is
+ * useful if your keyboard/keypad is missing keys and you want magic key support.
+ *
+ */
+
+/* key combination for magic key command */
+#define IS_COMMAND() ( \
```
Comment (drashna) [inline keyboards/ai03/orbit/config.h]: This define isn't actually needed, since it matches the default, and it should be removed.

**Comment by drashna** [comment]
Also, ideally, I'd rather see the split common code expanded to include this sort of functionality, rather than it being copied, and thus re-fragmenting the code.  (or at least, both done here)

File: keyboards/ai03/orbit/matrix.c (hunk 1 above)
Comment (ai03-2725) [reply keyboards/ai03/orbit/matrix.c]: Not sure, I didn't mess with the matrix aside from copying it straight from split_common.

File: keyboards/ai03/orbit/config.h (hunk 3 above)
Comment (ai03-2725) [reply keyboards/ai03/orbit/config.h]: Removed with new commits.

File: keyboards/ai03/orbit/config.h (hunk 2 above)
Comment (ai03-2725) [reply keyboards/ai03/orbit/config.h]: Removed with new commits.

**Comment by ai03-2725** [comment]
Replaced pin calls using QMK functions.

**Comment by drashna** [comment]
Thanks!

**Pull request merged**